README.md

```diff
@@ -158,6 +158,17 @@ Do not use English word abbreviations, such as `2nd`, `no.` and `approx.`. Use t
 
 Note that you are encouraged to use Latin abbreviations, such as `i.e.`, `e.g.`.
 
+### Evaluation Terminology
+
+- [Ratio](https://www.mathsisfun.com/numbers/ratio.html)
+    - Show that the number of times one value contains or is contained within the other.
+    - Formula: $A/B$
+- [Relative change / Relative difference / Percentage change](https://www.mathsisfun.com/numbers/percentage-change.html)
```

Comment:
I think we should list percentage difference as well, and highlight the difference, and explicitly mention that the percentage difference should not be used.

Reply:
For stackoverflow, people usually quote the content from other sources directly in the post, in case the url becomes invalid. For here, we should mention all the necessary content, and give that url as a reference.

Reply:
We need to add a note about our preferred metrics. I suppose most of the time we use ratio and %change